Source/Core/Core/PowerPC/MMU.h

```diff
@@ -190,8 +190,19 @@ bool IsOptimizableGatherPipeWrite(u32 address);
 struct TranslateResult
 {
   bool valid;
```

Comment:
May want to consider directly initializing these to false/0 directly. It'll simplify the implementations of two of the constructors below.

Reply:
Reasonable, yes.